### PR TITLE
WindowsPerf VS extension installation guide

### PR DESCRIPTION
**Pull request by nader-zouaoui**
Before submitting a pull request for a new Learning Path, please review [Create a Learning Path](https://learn.arm.com//learning-paths/cross-platform/_example-learning-path/)
- [x] I have reviewed Create a Learning Path

Please do not include any confidential information in your contribution. This includes confidential microarchitecture details and unannounced product information. No AI tool can be used to generate either content or code when creating a learning path or install guide.

- [x] I have checked my contribution for confidential information

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the Creative Commons Attribution 4.0 International License. 

-- 
This pull request includes updates to the contributors list and adds a comprehensive installation guide for the WindowsPerf Visual Studio Extension. The most important changes focus on documentation enhancements and updating contributor information.

Documentation Enhancements:

* [`content/install-guides/windows-perf-vs-extension.md`](diffhunk://#diff-71204ddec59da401241c53ced2a3fc005dbf65e209934aa71c1b773672de16e4R1-R91): Added a detailed installation guide for the WindowsPerf Visual Studio Extension, including prerequisites, installation steps, setup instructions, and a section on how to uninstall the extension.

Contributor Information Update:

* [`assets/contributors.csv`](diffhunk://#diff-69b030594bf841972794c9f52beee53684a563dac4df9905dde7d73860b9a460R44): Added Nader Zouaoui to the contributors list with relevant details.

related to https://github.com/arm-developer-tools/windowsperf-vs-extension/issues/11

### CODE REVIEW

File: content/install-guides/windows-perf-vs-extension.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+**WindowsPerf** is a lightweight performance profiling tool inspired by Linux perf, specifically tailored for Windows on ARM. It leverages the ARM64 PMU (Performance Monitor Unit) and its hardware counters to offer precise profiling capabilities. Recognizing the complexities of command-line interaction, the WindowsPerf GUI project is a Visual Studio 2022 extension that was initiated to provide a more intuitive, integrated within the development environment (IDE) experience. This tool enables developers to interact with WindowsPerf, adjust settings, and visualize performance data seamlessly within the IDE.
+
+## A Glimpse of the UI
+
+The WindowsPerf GUI extension is composed of several key features, each designed to streamline the user experience:
+
+![Sampling preview #center](../_images/wperf-vs-extension-sampling-preview.png)
+
+- **WindowsPerf Configuration**: Connect directly to `wperf.exe` for a seamless integration. Configuration is accessible via `Tools -> Options -> Windows Perf -> WindowsPerf Path`.
```
Comment (PrzemekWirkus) [inline content/install-guides/windows-perf-vs-extension.md:39]: I think you want to add here header like 

``## Installation`

This section is just cramped below the UI Glimpse picture.

image

Comment (PrzemekWirkus) [reply content/install-guides/windows-perf-vs-extension.md:39]: Hmmm, this is hard to understand. So this is the list of things it does?
Not sure if this is right way to do it, it's a bulletpoint list of things, I thought I should follow these.
Maybe it's better to just write a paragraph explaining these things?

File: content/install-guides/windows-perf-vs-extension.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+Unzip the downloaded file and double click on the `WindowsPerfGUI.vsix` file
+
+{{% notice Note %}}
+You need to make sure that any previous version of the extension is uninstalled and that Visual Studio is closed before installing the extension.
+{{% /notice %}}
+
+### Build from source
+
+For those preferring to build from the source, simply clone the repository and build the `WindowsPerfGUI` Project using the default configuration on Visual Studio. This alternative installation method offers flexibility and customization.
```
Comment (PrzemekWirkus) [inline content/install-guides/windows-perf-vs-extension.md:83]:
```suggestion
For those preferring to build from the source, simply clone [the repository](https://github.com/arm-developer-tools/windowsperf-vs-extension) and build the `WindowsPerfGUI` solution using the default configuration on Visual Studio. This alternative installation method offers flexibility and customization.
```

**Review by PrzemekWirkus — CHANGES_REQUESTED** [review]
Needs few improvements before Learning Path people review it.
But in general looks good!

**Comment by nader-zouaoui** [comment]
Hello @odincodeshen, I hope you are doing well!

This is an istallation guide for the WindowsPerf Visual Studio extension. I would appreciate it if you took a look at it

**Pull request merged**